.travis.yml

```diff
@@ -29,11 +29,11 @@ after_success:
     npm run coverage:upload;
   fi
   if [[ $TRAVIS_BRANCH == 'release' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
```

Comment:
Yes @edgesoft confirms, we should do the automatic release from "master". So this can be merged with the previous `if`.